.travis.rosinstall.kinetic

```diff
@@ -0,0 +1,78 @@
+- git:
+    uri: 'https://github.com/ipa320/cob_calibration_data.git'
+    local-name: cob_calibration_data
+    version: indigo_dev
+- git:
+    uri: 'https://github.com/ipa320/cob_android.git'
+    local-name: cob_android
+    version: indigo_dev
+- git:
+    uri: 'https://github.com/ipa320/cob_command_tools.git'
+    local-name: cob_command_tools
+    version: indigo_dev
+- git:
+    uri: 'https://github.com/ipa320/cob_common.git'
+    local-name: cob_common
+    version: indigo_dev
+- git:
+    uri: 'https://github.com/ipa320/cob_control.git'
+    local-name: cob_control
+    version: kinetic_dev
+- git:
+    uri: 'https://github.com/ipa320/cob_driver.git'
+    local-name: cob_driver
+    version: indigo_dev
+- git:
+    uri: 'https://github.com/ipa320/cob_environments.git'
+    local-name: cob_environments
+    version: indigo_dev
+- git:
+    uri: 'https://github.com/ipa320/cob_extern.git'
+    local-name: cob_extern
+    version: indigo_dev
+- git:
+    uri: 'https://github.com/ipa320/cob_hand.git'
+    local-name: cob-hand
+    version: indigo_dev
+- git:
+    uri: 'https://github.com/ipa320/cob_manipulation.git'
+    local-name: cob_manipulation
+    version: kinetic_dev
+- git:
+    uri: 'https://github.com/ipa320/cob_navigation.git'
+    local-name: cob_navigation
+    version: indigo_dev
+- git:
+    uri: 'https://github.com/ipa320/cob_perception_common.git'
+    local-name: cob_perception_common
+    version: indigo_dev
+- git:
+    uri: 'https://github.com/ipa320/cob_substitute.git'
+    local-name: cob_substitute
+    version: indigo_dev
+- git:
+    uri: 'https://github.com/ipa320/cob_supported_robots.git'
+    local-name: cob_supported_robots
+    version: indigo_dev
+- git:
+    uri: 'https://github.com/ipa320/schunk_modular_robotics.git'
+    local-name: schunk_modular_robotics
+    version: indigo_dev
+
```

Comment:
I used the blank line to separate repositories which we want to use from release but currently are not released for kinetic yet...it's kind of "the foreign" packages...i.e. not `cob_` packages